src/Microsoft.Diagnostics.Runtime/src/Implementation/ClrmdArrayType.cs

```diff
@@ -86,5 +86,30 @@ public override ulong GetArrayElementAddress(ulong objRef, int index)
 
             return ValueReader.GetValueAtAddress(Heap, DataReader, cet, addr);
         }
+
+        public override T[]? GetArrayElementsValues<T>(ulong objRef, int count)
+        {
+            ulong addr = GetArrayElementAddress(objRef, 0);
```

Comment:
Check if `addr == 0` and return null here as an early out.

Reply:
Yeah, sounds reasonable.